src/entities/campaign/components/CampaignBanner.tsx

```
@@ -107,42 +107,61 @@ export const CampaignBanner: React.FC<CampaignBannerProps> = ({ campaignId }) =>
                   <BadgeIcon size={5} />
                   <span className="m-0 font-bold">OFFICIAL</span>
                 </div>
               )}
             </div>
           </div>
         </div>
-        <p className="p-6">{campaign?.description}</p>
+
+        <div
+          className="prose prose-sm max-w-none"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)